src/components/Accordion/Accordion.stories.tsx

```diff
@@ -57,23 +61,48 @@ const Template: ComponentStory<typeof Accordion> = (args) => {
   const AccordionExampleContent =
     'Accordion-innhold uten css for å tilrettelegge for selvvalgt styling';
 
-  const ActionButton = <Button>Separat funksjonsknapp</Button>;
+  const [{ useActions }] = useArgs();
+  const [{ subtitle }] = useArgs();
+
+  const ActionDiv = useActions ? (
+    <div>
```

Comment:
To ensure that the styles from `accordion-header__actions` are applied, this should return a fragment instead `<>{content}</>`.

This gives the flexibility that by default, the items will be laid out with grid and with some space between them. If you add a single child to `actions`, you can control the layout yourself.

Reply:
Agreed 😊